cloud/deployment/fromfile/fromfile.go

```diff
@@ -198,8 +198,8 @@ func getCreateOrUpdateInput(deploymentFromFile *inspect.FormattedDeployment, clu
 			// check if more than one queue is requested
 			if len(listQueues) > 1 {
 				return astro.CreateDeploymentInput{}, astro.UpdateDeploymentInput{},
-					fmt.Errorf("%w more than one worker queue. (%d) were requested",
-						workerqueue.ErrNotSupported, len(listQueues))
+					fmt.Errorf("%s %w more than one worker queue. (%d) were requested",
+						"KubernetesExecutor", workerqueue.ErrNotSupported, len(listQueues))
```

Comment:
You can use `deployment.KubeExecutor` instead of defining it as string

Reply:
There are few more such occurence in this PR. Can you update the same?

Thinking forward, we should have a constant file something like `airflow/constants.go` so we could reference there instead of importing and referencing across different packages. Doesnt need to be addressed in this PR, but eventually we should be more consistent so the code is more readable

Reply:
@kushalmalani changed all the strings to use constants.